invenio_communities/ext.py

```diff
@@ -65,11 +66,20 @@ def init_app(self, app):
         @app.context_processor
         def record_context_processors():
             """Jinja context processors for communities record integration."""
-            def record_communities(record):
+            def communities_ui_data(record):
```

Comment:
This name is pretty generic, even though the function ends up being used only in the record view's JS app.